docs/docsource/operation/XDM_GUI/bucket_operations/index.rst

```diff
@@ -3,8 +3,8 @@
 Bucket Operations
 =================
 
-You can use the |product| UI to create, configure, access, and delete buckets
-from all locations under your control.
+You can use the Zenko UI to create, configure, access, and delete buckets from
```

Comment:
Was it intentional to go from product to Zenko?

Reply:
No.